plugins/module_utils/network/sonic/config/interfaces/interfaces.py

```diff
@@ -192,18 +193,23 @@ def _state_overridden(self, want, have, diff):
                   to the desired configuration
         """
         commands = []
-        commands_del = self.filter_comands_to_change(want, have)
-        requests = self.get_delete_interface_requests(commands_del, have)
-        del_req_count = len(requests)
-        if commands_del and del_req_count > 0:
-            commands_del = update_states(commands_del, "deleted")
-            commands.extend(commands_del)
-
-        commands_over = diff
-        requests.extend(self.get_modify_interface_requests(commands_over, have))
-        if commands_over and len(requests) > del_req_count:
-            commands_over = update_states(commands_over, "overridden")
-            commands.extend(commands_over)
+        requests = []
+
+        diff1 = get_diff(have, want)
+        if not diff and not diff1:
+            return commands, requests
+
+        commands = have
+        requests = self.get_delete_all_interface(commands, have)
+        if commands and len(requests) > 0:
+            send_requests(self._module, requests)
+        else:
+            commands = []
+        commands_over = get_diff(want, have)
+        requests = self.get_modify_interface_requests(diff, have)
```

Comment:
Line#202 to #207 deletes whole have. If want is subset of have, then diff is empty and requests is empty, so that there is nothing to apply.